packages/calypso-e2e/src/lib/pages/themes-page.ts

```diff
@@ -58,28 +58,42 @@ export class ThemesPage extends BaseContainer {
 	 */
 	async search( keyword: string ): Promise< void > {
 		const searchInput = await this.page.waitForSelector( selectors.searchInput );
-		await searchInput.fill( keyword );
+		await Promise.all( [ this.page.waitForNavigation(), searchInput.fill( keyword ) ] );
```

Comment:
Similar to my other question, do we want to wait for navigation specifically _after_ we are done filling in the search keyword? I.e.m moving the `await this.page.waitForNaviation()` outside the `Promise.all()`?

Reply:
This is a [fairly common](https://playwright.dev/docs/navigations) pattern in Playwright:
```
await Promise.all([
    wait for navigation
    something that triggers navigattion
])

This is because entering text in the Themes search box triggers a navigation. 

By placing `waitForNavigation` prior to the action, Playwright begins observing for a navigation and this is fulfilled when the action fires next. This can be observed in the logs that show what the Playwright API is doing in the background:
```
  pw:api => page.waitForNavigation started +1ms
  pw:api => elementHandle.fill started +2ms
  pw:api waiting for navigation until "load" +3ms
  pw:api elementHandle.fill("Twenty Twen") +1ms
  pw:api   waiting for element to be visible, enabled and editable +3s
  pw:api   element is visible, enabled and editable +17ms
  pw:api <= elementHandle.fill succeeded +12ms
  pw:api   navigated to "https://wordpress.com/themes/free/e2eflowtesting3.wordpress.com?s=Twenty+Twen" +331ms
  pw:api <= page.waitForNavigation succeeded +3ms
```

In this specific case here, I grouped the `fill` and `waitForNavigation` as a way to reduce reliance on selectors when waiting for actions on the page to be completed.